.dev/go.make

```diff
@@ -0,0 +1,32 @@
+# Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+GO_PACKAGES := $(shell go list -f '{{.Dir}}' ./...)
+
+.PHONY: go.fmt go.unconvert go.lint go.binaries go.test
+
+go.fmt:
+	gofmt -w -s $(GO_PACKAGES)
+
+go.unconvert:
+	go run github.com/mdempsky/unconvert -apply -safe $(GO_PACKAGES)
+
+go.lint:
+	go run github.com/mgechev/revive -config=.revive.toml -formatter=stylish $(GO_PACKAGES)
+
+go.binaries:
+	go run ./cmd/ttn-lw-migrate -h
+
+go.test:
+	go test ./... -timeout=5m
```

Comment:
`-failfast` ?

Reply:
Should we `-failfast` here? Is anything cached automagically? I think it is better not to, so that multiple errors are caught in a single CI run.

Reply:
If CI fails I'd prefer knowing immediately instead of waiting for it to finish other tests. Developers are expected to run tests locally for debugging purposes, we should not be wasting computing resources.
FWIW, I never run tests without `-failfast` even locally, but that's just preference. Please do what you think is best here.